src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -7,20 +7,19 @@
 ;; Schemas
 
 (def Type (schema/enum :string :regex))
-(def Method (schema/enum :get :post :put :delete :head :any))
+(def Methods (schema/enum :get :post :put :delete :head :any))
+(def Method (schema/either Methods [Methods]))
```

Comment:
I'd agree that the names seem backward.

>  just refer to multiples as [Method] rather than a dedicated schema for it

It would seem onerous to have to modify each of the locations where `method` is passed in as a parameter to do the `schema/either` allowing for one method or a list of methods.  I could see naming the schema that either takes one or a vector of them as `Methods` with `Method` as the schema that has the possible enum values.  I'm not too fussed about how we do this, though.